deployments/aws/lb-connectors/terraform.tfvars.sample

```diff
@@ -49,7 +49,7 @@ cac_instance_count_list = []
 admin_ssh_pub_key_file = "~/.ssh/id_rsa.pub"
 
 # REQUIRED: ensure the AWS account used has subscribed to the "NVIDIA Quadro
-# Virtual Workstation - WinServer 2016" image using the AWS dashboard at
+# Virtual Workstation - WinServer 2019" image using the AWS dashboard at
 # https://aws.amazon.com/marketplace/pp/B07TS3S3ZH
```

Comment:
I'll take these lines out in my other branch (since we no longer need to subscribe to this image)